src/serializer.ts

```diff
@@ -280,7 +280,14 @@ export class Serializer {
     if (node.properties.length) {
       data += '\n';
 
-      for (const property of node.properties) {
+      const orderedNodes = node.properties.sort((a, b) => {
```

Comment:
Random note from a random person randomly taking a look, but this
```javascript
      const orderedNodes = node.properties.sort((a, b) => {
```
looks like the expectation is to create a new sorted array, when it actually doesn't; `.sort` mutates the original. I'm not sure if it's a problem here in practice, but thought I'd point that out since it looks accidental here and could in some cases cause weird bugs elsewhere if the array is expected to be in some other order there.

Reply:
Hey, I see no problem; the tests prove the logic is working as intended. Before this change, no ordering check was being done on the nodes themselves anywhere. 

While you are correct that [sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) modifies the array in place, it also returns the sorted array and I'm simply assigning the sorted array to a new variable for easier handling.